services/director/source/producer.py

```diff
@@ -78,6 +90,16 @@ def __convert_labels_to_docker_runtime_parameters(service_runtime_parameters_lab
     _LOGGER.debug("Converted labels to docker runtime parameters: %s", runtime_params)
     return runtime_params
 
+def __get_service_entrypoint(service_boot_parameters_labels):
+    _LOGGER.debug("Getting service entrypoint")
+    for param in service_boot_parameters_labels:
+        if 'name' not in param or 'type' not in param or 'value' not in param:
```

Comment:
Thats a bit silly.

Reply:
replaced by a function. don't know if this is less silly ;)
it now raise an exception. the idea is to detect whether the labels are in order or not.